apiary.apib

```diff
@@ -990,6 +990,199 @@ Outputs the list of peers that were removed.
 
 # Group id
 
+Prints out information about the specified peer,
+if no peer is specified, prints out local peers info.
+
+'ipfs id' supports the format option for output with the following keys:
+<id> : the peers id
+<aver>: agent version
+<pver>: protocol version
+<pubkey>: public key
+<addrs>: addresses (newline delimited)
+
+## id [GET /id{?arg}{&format}]
```

Comment:
Wouldn't it be better if the format is something that the cli understand and transforms the json coming from the API?